shell/pages/auth/verify.vue

```diff
@@ -25,21 +25,21 @@ function isSaml($route) {
 }
 
 export default {
-  async fetch({ store, route, redirect }) {
-    const code = route.query[GITHUB_CODE];
-    const stateStr = route.query[GITHUB_NONCE];
+  async fetch() {
```

Comment:
Comment only. Was wondering if something like below was better (cleaner diff) BUT the rest of the file uses `this.$` so that's a better pattern

```
  async fetch() {
    const { $store: store, $route: route, $router: router } = this;
```

Reply:
I'm honestly fine with either. If the team wanted to adopt destructing I'd be happy. I'd just like to be consistent with one, has that already started and I haven't noticed?

Reply:
It's in use in a fair number of places, but i'm not sure it's a universal either/or thing. I'd vote 100% fine to use `this.` in this case